modules/storages/spec/features/admin_storages_spec.rb

```diff
@@ -34,6 +34,8 @@
 RSpec.describe 'Admin storages',
                :js,
                :storage_server_helpers do
+  include Redmine::I18n
```

Comment:
Do we need this mixin? `I18n` should be available already- and I don't see any references here? 😕

Reply:
I added it for `format_time()`.
I saw it in a another spec like this. Let me know if there is a better way to use format_time

Reply:
Got you 👍🏾  since we have a `frozen_date_time` we could assert against the generated string as it should ways be consistent- and yields more accurate tests